Directory.Build.targets

```diff
@@ -81,6 +81,7 @@
     <PackageDescription Condition="'$(PackageDescription)' == '' and '$(UseRuntimePackageDisclaimer)' == 'true'">$(RuntimePackageDisclaimer)</PackageDescription>
     <!-- Keep in sync as required by the Packaging SDK in Arcade. -->
     <Description>$(PackageDescription)</Description>
+    <!-- TMP: The next two lines were added a year ago and were never removed or changed: -->
     <!-- Until we update the TFM to match the Major version, we need to patch in support for RequiresPreviewFeatures -->
     <GenerateRequiresPreviewFeaturesAttribute Condition="'$(TargetFrameworkMoniker)' == '$(NetCoreAppCurrentTargetFrameworkMoniker)'">true</GenerateRequiresPreviewFeaturesAttribute>
```

Comment:
We never changed this even though the comment seemed to indicate the change was temporary, so I added the extra comment to show it in the PR.

Reply:
```suggestion
    <!-- During branding updates, the TFM may not match the Major version. In this case, we will need to patch in support for RequiresPreviewFeatures. -->
    <GenerateRequiresPreviewFeaturesAttribute Condition="'$(TargetFrameworkMoniker)' == '$(NetCoreAppCurrentTargetFrameworkMoniker)'">true</GenerateRequiresPreviewFeaturesAttribute>
```